hosting_profitability.md

```diff
@@ -1,19 +1,27 @@
+ 
+# How miners determine the potential revenue?
+There are three types of mining available:
 
-# How miner determine the potential revenue?
+- A CML state machine validator mining. A CML nodes have the important task of maintaining the TEA Project's state machine. Because of its importance, very few if any ordinary users will be able to run an A CML node for at least the first 2 years post mainnet launch. A CML earn TEA rewards at the rate shown in [[Staking_to_state_machine]].
+- B CML app hosting mining. When a user says they want to mine on the TEA Project, they will probably be running a B CML node. In addition to TEA public service rewards, B CML earn a share of the TApp tokens of the TApps they host as most TApp developers will choose TApp tokens as the hosting payout model.
+- C CML availability attestation mining. C CML mining is earned through making sure B CML nodes are online. Since there's only a modest reward for performing this public service, we won't discuss C CML mining as a viable form of profitable mining in this document.
 
 # Other types of profitabilities in TEA
-## Staking to a CML
-Staking to CML is .....TODO...
-Link to [[staking_to_a_CML]]
+There are other forms of staking and mining available to earn revenue on the TEA network. Note that all of the staking tokens listed below are issued along a bonding curve where an increase supply results in an increase in price.
+
+## Staking to B CML mining nodes
+Staking to B CML nodes is a way to earn a share of mining revenue without having to setup a mining machine. Users will buy miner staking tokens for an individual B mining node in order to share in a mining machine's rewards, either as TEA revenue or TApp token dividend rewards. More info is available at [[staking_to_b_CML]].
 
 ## Staking to state machine
-TODO...
-Link to [[staking to state machine]]
-## Staking to a TApplication
-Linke to [[staking_to_tapp]]
-## Layer one (blockchain) validator
-This only works to the TEA native layer1. The one written in Substrate. There is no layer1 validator when using the Ethereum and other EVM-based blockchain as layer1.
+Similar to B CML mining nodes, A CML state machine validators will have a staking token that users can buy to earn a share of the validator nodes' rewards. Unlike with B CML staking, the state machine validator mining stake token represents a share in the profits of all state machine validators collectively. More info is available at [[staking to state machine]].
 
-...TODO...
+## Staking to a TApplication (TApp)
+To participate in the revenue generated by a TApp, a user can buy that TApp's tokens. Every time a consumer uses a TApp, the developer gets their **theta** percentage share of the consumer's TEA payment and the rest is injected into its TApp token's bonding curve. The new TApp tokens that are generated from this event are distributed proportionally to existing TApp token holders. More info is available at [[staking_to_tapp]].
```

Comment:
this idea is still under discussion.
I was thinking to remove theta to simplify the concept.
Well, we can keep it as is and discuss then update